### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by joshualyon7**

### CODE REVIEW

File: src/pieces/commandManager.ts [hunk 1]
```diff
@@ -88,7 +88,8 @@ async function runCommand(msg: Message) {
 
 	if (msg.channel.type === 'DM' && command.runInDM === false) return msg.reply(`${command.name} is not available in DMs.`);
 	if (msg.channel.type === 'GUILD_TEXT' && command.runInGuild === false) {
-		await msg.reply('That command is not available here. Try again in DMs.');
+		await msg.author.send('The command you just tried to run is not available in public channels. Try again in DMs')
```
Comment (BenSegal855) [inline src/pieces/commandManager.ts:91]: Do you want to add a ping here to emulate v2.2.x?

**Comment by BenSegal855** [comment]
Oh you also may want to update the :stevepeace: emoji in this PR for the next minor update.

File: src/pieces/commandManager.ts (hunk 1 above)
Comment (joshualyon7) [reply src/pieces/commandManager.ts:91]: I don't think I know what you mean by that

Comment (BenSegal855) [reply src/pieces/commandManager.ts:91]: Something like this.
```suggestion
		await msg.author.send(`<@!${msg.author.id}>, The command you just tried to run is not available in public channels. Try again in DMs.`)
```

**Review by renross — APPROVED** [review]

**Pull request merged**